backend/app/usecase/service/keyfetcher.go

```diff
@@ -1,8 +1,9 @@
 package service
 
-import "github.com/short-d/kgs/app/entity"
+// Key represents unique identifier
+type Key string
```

Comment:
added a Key type, so that we don't rely on external package

Reply:
Should this be part of entity?

Reply:
according to our offline discussion, this should be put in service layer, which is right here

Reply:
Discussed offline. Keeping Key in service package for now.